src/components/ec-smart-table/ec-smart-table.vue

```diff
@@ -173,6 +175,14 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+  isPageSizeHidden: {
+    type: Boolean,
+    default: () => undefined,
```

Comment:
Why we need 3 states for these booleans everywhere? Use true/false to render the page size part and then media query in CSS to hide it/show it when on small screens. If it was rendered, it will hide/show, if it's not rendered, then it will be ignored.

Reply:
Three states was to have possibility force component to show/hide(whatever component is doing regarding the media query) and undefined for the default behaviour.